### PR TITLE
Updated typescript types to include all item variants

### PR DESCRIPTION
**Pull request by lededje**
Items can be either an array of strings or a label-value object

### CODE REVIEW

File: react-native-picker-module.d.ts [hunk 1]
```diff
@@ -5,9 +5,14 @@
 
 import React from "react"
 
+export type PickerItem<V = any> = string[] | {
```
Comment (julian-kingman-lark) [inline react-native-picker-module.d.ts:8]: Aren't there a limited number of items value can be? I would think only `number` or `string`

**Comment by julian-kingman-lark** [comment]
are there any plans on merging this?

File: react-native-picker-module.d.ts (hunk 1 above)
Comment (lededje) [reply react-native-picker-module.d.ts:8]: Yes, you are correct. I don't use this library any more, looks like it's no longer maintained

**Comment by JulianKingman** [comment]
@talut if this PR were fixed, would you merge it?

**Pull request closed without merging**